core/src/task.rs

```diff
@@ -632,30 +632,30 @@ pub enum AuthenticationToken {
     /// [2]: https://datatracker.ietf.org/doc/html/draft-dcook-ppm-dap-interop-test-design-03#section-4.3.3
     /// [3]: https://datatracker.ietf.org/doc/html/draft-dcook-ppm-dap-interop-test-design-03#section-4.4.2
     /// [4]: https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap-01#name-https-sender-authentication
-    DapAuth(TokenInner),
+    DapAuth(DapAuthTokenInner),
```

Comment:
Consider just having these be `BearerToken` and `DapAuthToken`. The `Inner` suffix suggests that there's some different public opaque outer type, which is not the case here.